packages/itwinui-css/src/text/skeleton.scss

```diff
@@ -17,6 +17,10 @@
   );
   background-size: 200% 100%;
 
+  svg {
+    fill: transparent;
```

Comment:
If an icon is placed within a `<h1>`, `<p>`, etc. this hides the SVG so it doesn't show up within the skeleton text.